CleanCode.StringKata/StringCalculator.cs

```diff
@@ -8,9 +8,137 @@ namespace CleanCode.StringKata
 {
     public class StringCalculator
     {
-        public int Calculate(string input)
+        private List<string> _separators = new List<string>();
+
+        public int Add(string input)
+        {
+            if (IsStringEmpty(input))
+            {
+                return HandleEmptyString();
+            }
+
+            return HandleString(input);
+        }
+
+        private int HandleString(string input)
+        {
+            string[] elements = ExtractElements(input);
+
+            return HandleElements(elements);
+        }
+
+        private int HandleElements(string[] elements)
+        {
+            List<int> numbers = ConvertToNumbers(elements);
+
+            ValidateNumbers(numbers);
+
+            numbers = FilterOutTooBigNumbers(numbers);
```

Comment:
Warto by tę metodą sparametryzować. Np. FilterOutNumbersGreaterThan(numbers, 1000); ale jak popatrzymy to jest to jednolinijkowiec gdzie wywołujemy Linq Where - co samo z siebie jest czytelne.